internal/display/bufwindow.go

```diff
@@ -106,51 +106,35 @@ func (w *BufWindow) Clear() {
 	}
 }
 
-// Bottomline returns the line number of the lowest line in the view
-// You might think that this is obviously just v.StartLine + v.Height
-// but if softwrap is enabled things get complicated since one buffer
-// line can take up multiple lines in the view
-func (w *BufWindow) Bottomline() int {
```

Comment:
`Bottomline` removed because its use is buggy anyway: if the bottom line is wrapped, `Bottomline` doesn't tell which row is at the bottom.
Instead of `w.Bottomline()` use `w.Scroll(v.StartLine, height-1)`.

Reply:
BTW this `Bottomline` implementation is inconsistent: if we've scrolled past the last line, it returns different values with softwrap=off and softwrap=on (which results in different behavior).